tsconfig.json

```
@@ -1,7 +1,7 @@
 {
 	"compilerOptions": {
 		"esModuleInterop": true,
-		"outDir": "./dist",
+		"outDir": "./build",
```

let's keep it as `dist`
```suggestion
		"outDir": "./dist",
```